src/sentry/api/serializers/rest_framework/rule.py

```diff
@@ -184,13 +187,41 @@ def save(self, rule):
         return rule
 
 
+ACTION_UUID_KEY = "uuid"
```

Comment:
nit: uuid by itself is pretty clear

Reply:
This should actually probably from the from Rule module, so let me pull this into my other PR